mysql-test/suite/galera/r/galera_binlog_stmt_autoinc.result

```diff
@@ -0,0 +1,147 @@
+SET GLOBAL wsrep_forced_binlog_format='STATEMENT';
```

Comment:
There are some test failures when this is applied to current 10.1:

Completed: Failed 5/989 tests, 99.49% were successful.

Failing test(s): galera.galera_binlog_stmt_autoinc innodb.innodb-autoinc innodb.doublewrite

From these you may ignore innodb.doublewrite.

Reply:
I suggest using ./mtr --suite=main,innodb,wsrep,galera,percona --vardir=/dev/shm --parallel=8 --force

Here is the second regression:

innodb.innodb-autoinc 'xtradb'           w6 [ fail ]
        Test ended at 2018-07-30 09:14:38

CURRENT_TEST: innodb.innodb-autoinc
mysqltest: At line 15: query 'INSERT INTO t1 (c2) VALUES ('innodb')' failed with wrong errno 1062: 'Duplicate entry '9223372036854775807' for key 'PRIMARY'', instead of 167...

The result from queries just before the failure was:
drop table if exists t1;
CREATE TABLE t1 (c1 BIGINT PRIMARY KEY AUTO_INCREMENT, c2 VARCHAR(10)) ENGINE=InnoDB;
INSERT INTO t1 VALUES (9223372036854775807, null);
INSERT INTO t1 (c2) VALUES ('innodb');

Reply:
galera.galera_binlog_stmt_autoinc 'xtradb' w3 [ fail ]
        Test ended at 2018-07-30 09:13:11

CURRENT_TEST: galera.galera_binlog_stmt_autoinc
--- /home/jan/mysql/10.1-orig/mysql-test/suite/galera/r/galera_binlog_stmt_autoinc.result	2018-07-30 08:31:22.429570158 +0300
+++ /home/jan/mysql/10.1-orig/mysql-test/suite/galera/r/galera_binlog_stmt_autoinc.reject	2018-07-30 09:13:11.049235299 +0300
@@ -8,20 +8,20 @@
 insert into t1(i) values(null);
 select * from t1;
 i	c
-1	dummy_text
+4	dummy_text
 insert into t1(i) values(null), (null), (null);
 select * from t1;
 i	c
-1	dummy_text
-3	dummy_text
-5	dummy_text
-7	dummy_text
+4	dummy_text
+6	dummy_text
+8	dummy_text
+10	dummy_text
 select * from t1;
 i	c
-1	dummy_text
-3	dummy_text
-5	dummy_text
-7	dummy_text
+4	dummy_text
+6	dummy_text
+8	dummy_text
+10	dummy_text
 SET GLOBAL wsrep_forced_binlog_format='none';
 SET GLOBAL wsrep_forced_binlog_format='none';
 drop table t1;
@@ -40,20 +40,20 @@
 insert into t1(i) values(null);
 select * from t1;
 i	c
-1	dummy_text
+4	dummy_text
 insert into t1(i) values(null), (null), (null);
 select * from t1;
 i	c
-1	dummy_text
 4	dummy_text
 7	dummy_text
 10	dummy_text
+13	dummy_text
 select * from t1;
 i	c
-1	dummy_text
 4	dummy_text
 7	dummy_text
 10	dummy_text
+13	dummy_text
 SET GLOBAL wsrep_auto_increment_control='ON';
 SET SESSION binlog_format='ROW';
 show variables like 'binlog_format';
@@ -62,7 +62,7 @@
 show variables like '%auto_increment%';
 Variable_name	Value
 auto_increment_increment	2
-auto_increment_offset	1
+auto_increment_offset	2
 wsrep_auto_increment_control	ON
 SET GLOBAL wsrep_auto_increment_control='OFF';
 show variables like '%auto_increment%';
@@ -82,20 +82,20 @@
 insert into t1(i) values(null);
 select * from t1;
 i	c
-1	dummy_text
+4	dummy_text
 insert into t1(i) values(null), (null), (null);
 select * from t1;
 i	c
-1	dummy_text
-3	dummy_text
-5	dummy_text
-7	dummy_text
+4	dummy_text
+6	dummy_text
+8	dummy_text
+10	dummy_text
 select * from t1;
 i	c
-1	dummy_text
-3	dummy_text
-5	dummy_text
-7	dummy_text
+4	dummy_text
+6	dummy_text
+8	dummy_text
+10	dummy_text
 SET GLOBAL wsrep_forced_binlog_format='none';
 SET GLOBAL wsrep_forced_binlog_format='none';
 drop table t1;
@@ -114,20 +114,20 @@
 insert into t1(i) values(null);
 select * from t1;
 i	c
-1	dummy_text
+4	dummy_text
 insert into t1(i) values(null), (null), (null);
 select * from t1;
 i	c
-1	dummy_text
 4	dummy_text
 7	dummy_text
 10	dummy_text
+13	dummy_text
 select * from t1;
 i	c
-1	dummy_text
 4	dummy_text
 7	dummy_text
 10	dummy_text
+13	dummy_text
 SET GLOBAL wsrep_auto_increment_control='ON';
 show variables like 'binlog_format';
 Variable_name	Value
@@ -135,7 +135,7 @@
 show variables like '%auto_increment%';
 Variable_name	Value
 auto_increment_increment	2
-auto_increment_offset	1
+auto_increment_offset	2
 wsrep_auto_increment_control	ON
 SET GLOBAL wsrep_auto_increment_control='OFF';
 show variables like '%auto_increment%';

mysqltest: Result length mismatch

Reply:
Thanks, I removed these regressions